src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs

```diff
@@ -487,7 +490,7 @@ public void ILLink_verify_analysis_warnings_hello_world_app_trim_mode_copyused(s
             var testAsset = _testAssetsManager.CreateTestProject(testProject);
 
             var publishCommand = new PublishCommand(Log, Path.Combine(testAsset.TestRoot, testProject.Name));
-            var result = publishCommand.Execute($"/p:RuntimeIdentifier={rid}", $"/p:SelfContained=true", "/p:PublishTrimmed=true", "/p:SuppressTrimAnalysisWarnings=false");
+            var result = publishCommand.Execute($"/p:RuntimeIdentifier={rid}", $"/p:SelfContained=true", "/p:PublishTrimmed=true", "/p:SuppressTrimAnalysisWarnings=false", "/p:TrimMode=copyused");
```

Comment:
Can we remove `/p:SuppressTrimAnalysisWarnings=false` ?

Reply:
Same for the `ILLink_verify_analysis_warnings_hello_world_app_trim_mode_link` test below.

---
In reply to: [595275610](https://github.com/dotnet/sdk/pull/16327#discussion_r595275610) [](ancestors = 595275610)